.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__/
 package-lock.json
 npm-debug.log
 .eslintcache
+/.vscode
```

Comment:
shouldn't be here, this should be in your global `.gitignore`

Reply:
can't hurt though? :)